RiotSharp/Http/ResponseDeserializer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using RiotSharp.Http.Interfaces;
+
+namespace RiotSharp.Http
+{
+    public class ResponseDeserializer : IResponseDeserializer
+    {
+        public async Task<T> DeserializeToAsync<T>(HttpResponseMessage message)
+        {
+            var json = await message.Content.ReadAsStringAsync();
+            return JsonConvert.DeserializeObject<T>(json);
```

Comment:
Couldn't you run this in a task?

Reply:
I did not see the benefit. After a bit of googling I have changed my mind and will put run it in one. Thanks